app/data/models/delinkedCalculation.js

```diff
@@ -16,30 +16,33 @@ module.exports = (sequelize, DataTypes) => {
     percentageReduction2: { type: DataTypes.STRING, allowNull: false },
     percentageReduction3: { type: DataTypes.STRING, allowNull: false },
     percentageReduction4: { type: DataTypes.STRING, allowNull: false },
-    progressiveReductions1: { type: DataTypes.STRING, allowNull: false },
-    progressiveReductions2: { type: DataTypes.STRING, allowNull: false },
-    progressiveReductions3: { type: DataTypes.STRING, allowNull: false },
-    progressiveReductions4: { type: DataTypes.STRING, allowNull: false },
-    referenceAmount: { type: DataTypes.STRING, allowNull: false },
-    totalProgressiveReduction: { type: DataTypes.STRING, allowNull: false },
-    totalDelinkedPayment: { type: DataTypes.STRING, allowNull: false },
+    progressiveReductions1: { type: DataTypes.STRING, allowNull: true },
+    progressiveReductions2: { type: DataTypes.STRING, allowNull: true },
+    progressiveReductions3: { type: DataTypes.STRING, allowNull: true },
+    progressiveReductions4: { type: DataTypes.STRING, allowNull: true },
+    referenceAmount: { type: DataTypes.STRING, allowNull: true },
+    totalProgressiveReduction: { type: DataTypes.STRING, allowNull: true },
+    totalDelinkedPayment: { type: DataTypes.STRING, allowNull: true },
```

Comment:
I don't actually know the answer to this so I'll defer to your knowledge - do we need an `allowNull: true`? How does adding that differ from datePublished which has no allowNull prop?

Reply:
updated to include allowNull: true